src/Integration.Vsix.UnitTests/CFamily/CFamilyIssueToAnalysisIssueConverterTests.cs

```diff
@@ -448,6 +448,8 @@ public void Convert_SeverityAndTypeLookup(string ruleKey, AnalysisIssueSeverity
 
         [TestMethod]
         [Description("Regression test for https://github.com/SonarSource/sonarlint-visualstudio/issues/2149")]
+        [DataRow("", "")] // empty should not throw
```

Comment:
We had a bit of an odd logic previously. We handled non-rooted paths for the primary location, but not the secondary. I think it was an oversight, or perhaps it was something that was not "supposed" to happen anyway.
Either way, there is now once single method that parses locations so the logic would be identical to both primary and secondary, hence I added these test cases to document explicitly what would happen.